charts/alertmanager/templates/statefulset.yaml

```
@@ -28,6 +28,14 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          args: 
+            - --storage.path=/alertmanager
+            {{- if .Values.configFileName }}
+            - --config.file=/etc/alertmanager/{{ .Values.configFileName }}
```

```suggestion
            - --config.file=/etc/alertmanager/{{ .Values.configFileName }}
```

👏 
As per other comments I think we can leave the filename hardcoded.

---

ok